### PR TITLE
Added overloads without the HostingContext

### PR DESCRIPTION
**Pull request by davidfowl**
Fixes https://github.com/aspnet/Hosting/issues/1219

### CODE REVIEW

File: src/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.cs [hunk 1]
```diff
@@ -62,6 +62,53 @@ public static IHostBuilder ConfigureLogging(this IHostBuilder hostBuilder, Actio
...
+        }
+        /// <summary>
+        /// Sets up the configuration for the remainder of the build process and application. This can be called multiple times and
+        /// the results will be additive. The results will be available at <see cref="HostBuilderContext.Configuration"/> for
+        /// subsequent operations, as well as in <see cref="IHost.Services"/>.
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="IHostBuilder" /> to configure.</param>
+        /// <param name="configureDelegate"></param>
+        /// <returns>The same instance of the <see cref="IHostBuilder"/> for chaining.</returns>
+        public static IHostBuilder ConfigureAppConfiguration(this IHostBuilder hostBuilder, Action<IConfigurationBuilder> configureDelegate)
```
Comment (Tratcher) [inline src/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.cs]: These are generic enough they can move to Ext.Hosting.Abstractions.

File: src/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.cs [hunk 2]
```diff
@@ -62,6 +62,53 @@ public static IHostBuilder ConfigureLogging(this IHostBuilder hostBuilder, Actio
...
+            return hostBuilder.ConfigureAppConfiguration((context, builder) => configureDelegate(builder));
+        }
+
+        /// <summary>
+        /// Adds services to the container. This can be called multiple times and the results will be additive.
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="IHostBuilder" /> to configure.</param>
+        /// <param name="configureDelegate"></param>
+        /// <returns>The same instance of the <see cref="IHostBuilder"/> for chaining.</returns>
+        public static IHostBuilder ConfigureServices(this IHostBuilder hostBuilder, Action<IServiceCollection> configureDelegate)
```
Comment (Tratcher) [inline src/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.cs]: Might as well use these in a few of the tests.

File: src/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.cs (hunk 1 above)
Comment (davidfowl) [reply src/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.cs]: I didn't move them because all can't be moved without adding more deps.

Comment (Tratcher) [reply src/Microsoft.Extensions.Hosting/HostingHostBuilderExtensions.cs]: I didn't say all, just most of your new ones.

**Pull request merged**